src/styles/block/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1px;
+  width: 725px;
+  margin: auto;
+  margin-top: 35px;
+}
+
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  padding: 49px;
+  border: 1px solid;
+  background-color: #eee;
+  transition: transform 0.5s;
+
+  @for $index from 29 through 31 {
```

Comment:
this loop doesn't work, need to fix)
U are using class for calendar, but don't do it in loop